fix for ng-html-bind directive handling

myukselen opened the pull request:
There were two problems:

* constant strings having &quot;
* ng-html-bind ternary regexp version matching beyond attribute boundary

myukselen committed:
fix(regexp): handle ng-html-bind constant string and ternary form

when ng-html-bind includes a constant string with &quot;, it was not converted to double quotes.

ng-html-bind ternary regex is hopefully fixed also

myukselen committed:
refactor(regexp): delete forgotten debug line

In order to find erronous regexp, a debug log section was added. This removes it.

review-ninja commented:
<a href="https://app.review.ninja/Boulangerie/angular-translate-extract/pull/13" target="_blank"><img src="https://app.review.ninja/52343395/pull/13/badge" alt="ReviewNinja"/>

myukselen committed:
fix(interpolation): fixed escaped interpolation string handling

Interpolation string should be escaped properly when used in an interpolation. This is handled in
HtmlFilterDoubleQuote and HtmlFilterSimpleQuote translationKey processing.
